app/serializers/poster_serializer.rb

```diff
@@ -23,4 +12,22 @@ def self.format_poster(poster)
         img_url: poster.img_url }
     }
   end
+  
+  def self.format_posters(posters)
+    # binding.pry
```

Comment:
Can we remove the pry? :-)

Reply:
Pry has been removed, pull request has been updated.